src/index.ts

```diff
@@ -26,26 +26,41 @@ export function getClient<TMode extends PollingMode | undefined>(sdkKey: string,
 }
 
 /**
- * Disposes all existing ConfigCatClient instances.
+ * Disposes all existing `ConfigCatClient` instances.
  */
 export function disposeAllClients(): void {
   configcatcommon.disposeAllClients();
 }
 
+/**
+ * Creates an instance of `ConfigCatConsoleLogger`.
+ * @param logLevel Log level (the minimum level to use for filtering log events).
+ */
 export function createConsoleLogger(logLevel: LogLevel): IConfigCatLogger {
   return configcatcommon.createConsoleLogger(logLevel);
 }
 
+/**
+ * Creates an instance of `FlagOverrides` that uses a map data source.
+ * @param map The map that contains the overrides.
+ * @param behaviour The override behaviour.
+ * Specifies whether the local values should override the remote values
+ * or local values should only be used when a remote value doesn't exist
+ * or the local values should be used only.
+ */
 export function createFlagOverridesFromMap(map: { [name: string]: NonNullable<SettingValue> }, behaviour: OverrideBehaviour): FlagOverrides {
   return new FlagOverrides(new MapOverrideDataSource(map), behaviour);
 }
 
+/** Options used to configure the ConfigCat SDK in the case of Auto Polling mode. */
```

Comment:
```suggestion
/** Options used to set up the ConfigCat SDK in the case of Auto Polling mode. */
```

Reply:
We decided to keep the current wording for consistency with other SDKs.